hack/make/gccgo

```diff
@@ -8,6 +8,9 @@ BINARY_FULLNAME="$BINARY_NAME$BINARY_EXTENSION"
 
 source "$(dirname "$BASH_SOURCE")/.go-autogen"
 
+if [[ "${BUILDFLAGS[@]}" =~ "netgo" ]]; then
```

Comment:
minor nit, `netgo` here should be in single quotes instead, and probably include a trailing space like `dynbinary` does when replacing it (for consistency):
```bash
if [[ "${BUILDFLAGS[@]}" =~ 'netgo ' ]]; then
```